Makefile

```diff
@@ -45,22 +45,24 @@ install: check
 	@if [ "$(INSTALL_VIA)" = brew ]; then \
 		git apply brew-release.patch || { echo "Can't apply brew release patch"; exit 1; } \
 	fi
-	@mkdir -p $(DESTDIR)$(MANPREFIX)
-	@mkdir -p $(DESTDIR)$(BINPREFIX)
-	@echo "... installing bins to $(DESTDIR)$(BINPREFIX)"
-	@echo "... installing man pages to $(DESTDIR)$(MANPREFIX)"
+	mkdir -p $(DESTDIR)$(MANPREFIX)
```

Comment:
It seems that these cmds will be broken when the directories contain space. If it is true, would you also improve them in  this PR?

Reply:
From my testing it seems you're not supposed to quote variable interpolations like this because the values will automatically be escaped. that's why I changed the quotes on lines 50 and 51 - to no longer include `$(DESTDIR)$(BINPREFIX)`.

Reply:
Interestingly, single quotes are OK, but double quotes breaks things.

Looking at other popular projects, it seems the norm is not to quote things: https://github.com/torvalds/linux/blob/6465e260f48790807eef06b583b38ca9789b6072/Makefile#L1283, which seems to line up with other information on how to use GNU Make

Reply:
I see